net_utils.js

```diff
@@ -187,10 +180,59 @@ exports.is_ip_in_str = function(ip, str) {
     return false;
 };
 
-exports.is_rfc1918 = function (ip) {
-    return (net.isIPv4(ip) && re_private_ipv4.test(ip));
+exports.is_private_ipv4 = function (ip) {
+
+    // RFC 1918, reserved as "private" IP space
+    if (/^10\./.test(ip)) return true;          // 10/8
```

Comment:
I benchmarked just the is_private_ipv4() function above against your stock version through 1,000,000 iterations and I found the pre-compiled regexps to be on average 1 second faster.   I tested a static value that was always matched in both cases.

`````
var result;
for (var i=0; i<10000000; i++) {
    result = exports.is_private_ipv4('192.168.1.1');
}
`````